vendor/pischem/caddy/src/etc/inc/plugins.inc.d/caddy.inc

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * Copyright (C) 2023 Cedrik Pischem
+ * Copyright (C) 2017 EURO-LOG AG
```

Comment:
Copied from monit or relayd? You can remove this line because nothing from this original file is left.

Reply:
Yeah it was copied from monit . Ill remove it.